lib/cdo/redshift_import.rb

```diff
@@ -0,0 +1,170 @@
+require 'cdo/redshift'
+require 'pg'
+
+class RedshiftImport
+  # Database Migration Service Replication Tasks load data from Aurora into staging Redshift tables with a prefix.
+  TEMP_TABLE_PREFIX = '_import_'.freeze
```

Comment:
We should use this constant for the string in [`DMS#tasks`](https://github.com/code-dot-org/code-dot-org/pull/32992/files#diff-03480daff31b9911d52471c2e2ec0c73R52) as well